BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/BetterShardsListener.java

```diff
@@ -489,4 +493,31 @@ public void blockPlaceEvent(BlockPlaceEvent event) {
 			event.setCancelled(true);
 		}
 	}
+	
+	@EventHandler(priority = EventPriority.LOWEST)
+	public void portalLoadEvent(PortalLoadEvent event) {
```

Comment:
I believe it's only bad practice when you don't mention where to find that specific code which in fairness I didn't.  I was also not sure how I should do this.  I was thinking I could keep the code in the load method but then not call the event for BetterShards portals but then there is a state where the event is only called for other plugins making it inconsistent